projects/packages/analyzer/src/api/class-router.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Automattic\Jetpack\Analyzer;
+
+/**
+ * Router class. App entry point
+ */
+class Router {
+
+	private $status;
+	private $analyze;
+
+	public function __construct() {
+		$this->status  = new Status_Controller();
+		$this->analyze = new Analyze_Controller();
+	}
+
+	public function handle_request() {
+		if ( str_contains( $_SERVER['REQUEST_URI'], '?status' ) ) {
```

Comment:
This is only in PHP 8.0+. I'm fine enforcing 8.0 as a minimum for this package due to it's unique status (not for production sites, etc). I've excluded this package from the PHPCompatability rules and set in `composer.json` that PHP 8 is required ( 8d5cfae4c0958b947d793b6fdc5cc2d8479cb439 )

Reply:
You didn't set that PHP 8 is required, you set Composer to pretend that PHP 8 is what's installed when resolving packages regardless of what's actually running.

Enforcing PHP 8 would involve an entry in `require`, and also adjusting the `test-php` and `test-coverage` scripts to not actually run any tests if PHP <8 is used (like codesniffer does) so CI won't fail.

Or you could just change the `str_contains()` to something like `strpos() !== false`.